devicetypes/rooms-beautiful/curtain.src/curtain.groovy

```diff
@@ -0,0 +1,260 @@
+/**
+ *
+ *	Copyright 2019 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+
+import groovy.json.JsonOutput
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "Curtain", namespace: "Rooms Beautiful", author: "Alex Feng", ocfDeviceType: "oic.d.blind", mnmn: "SmartThings", vid: "generic-shade") {
```

Comment:
One last change: I am uploading some metadata that better aligns with your capability set. It will use the vid `generic-shade-4`. If you could please set to that then I think this will be all ready!

Reply:
@dkirker I've just committed the change.